### PR TITLE
Add brick:Relationship superproperty

### PR DESCRIPTION
**Pull request by gtfierro**
Rationale in discussion of #415 . I am adding this not for query performance but so that we have a parallel structure to `brick:Entity` (the root class of all Brick classes) for relationships. This *only* is for inter-entity relationships, so EntityProperties are not included under the `brick:Relationship` property hierarchy

This should be merged into a 1.3.1 should the change be accepted.

### CODE REVIEW

**Comment by ektrah** [comment]
Two quick questions:

* What's the relationship between the new `brick:Relationship` and the existing `brick:EntityProperty`?
* Would it make sense to call it `brick:Property` instead of `brick:Relationship`?

**Comment by gtfierro** [comment]
> What's the relationship between the new brick:Relationship and the existing brick:EntityProperty?

Entity Properties would be sibling to the proposed Relationship; Relationships are properties whose value is another "entity" (instance of Brick class). Entity Properties are static attributes of Brick entities.

> Would it make sense to call it brick:Property instead of brick:Relationship?

I'd definitely consider that. Only reason I went with Relationship is because that is the word we've historically used in Brick publications to talk about the relationship between Brick entities.

**Comment by gtfierro** [comment]
@jbkoh thoughts?

**Comment by epaulson** [comment]
We've been using 'relationship' for a long, long time. I wouldn't be excited about changing that term.

**Comment by jbkoh** [comment]
I like Relationship and also its distinction from EntityProperty, which is common in property graphs.

File: tests/test_properties.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import brickschema
+from brickschema.namespaces import BRICK, RDFS
+import sys
+
+sys.path.append("..")
+from bricksrc.properties import properties  # noqa:E402
```
Comment (jbkoh) [inline tests/test_properties.py:6]: Maybe we can change the file name and the object name to relationships from now on to be explicit on the naming? It's not critical but would be nice to have.

**Review by jbkoh — CHANGES_REQUESTED** [review]
The pedantic side of me wants to change the file name from properties.py to relationships.py to be aligned with the new names though it's not critical. I will let you decide it.

File: tests/test_properties.py (hunk 1 above)
Comment (gtfierro) [reply tests/test_properties.py:6]: Nice idea! I'll see how easy it is to make that adjustment

**Pull request merged**